[2022-07-18 23:52:04,958] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2012-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:52:04,967] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2012-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:52:04,968] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:52:04,970] {taskinstance.py:1358} INFO - Starting attempt 3 of 3
[2022-07-18 23:52:04,971] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:52:04,987] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2012-01-10 06:00:00+00:00
[2022-07-18 23:52:04,995] {standard_task_runner.py:52} INFO - Started process 40456 to run task
[2022-07-18 23:52:05,001] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_gcs', 'local_to_gcs_task', 'scheduled__2012-01-10T06:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/local_to_gcs.py', '--cfg-path', '/tmp/tmpeiyu3zdi', '--error-file', '/tmp/tmptek6nk_h']
[2022-07-18 23:52:05,003] {standard_task_runner.py:80} INFO - Job 85: Subtask local_to_gcs_task
[2022-07-18 23:52:05,076] {task_command.py:370} INFO - Running <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2012-01-10T06:00:00+00:00 [running]> on host 32d1f26135d5
[2022-07-18 23:52:05,193] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_gcs
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2012-01-10T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2012-01-10T06:00:00+00:00
[2022-07-18 23:52:13,512] {python.py:173} INFO - Done. Returned value was: None
[2022-07-18 23:52:13,578] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=local_to_gcs, task_id=local_to_gcs_task, execution_date=20120110T060000, start_date=20220718T235204, end_date=20220718T235213
[2022-07-18 23:52:13,756] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-18 23:52:13,833] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
